### PR TITLE
Issue 703 - Navigation Links the Nextjs way

### LINKED ISSUE #703: Links in main navigation are causing full page reload
Should be being handled by the router.

### PR DESCRIPTION
**Pull request by dashcraft**
Fixes #703  navigation links, there where some other consistency updates:
- Navigation Links are handled by nextjs router/link component
- Styles for navbar made consistent during loading so it's not flashing white
- Added the maintenance banner to loading so the navigation bar doesn't move when loading
- Updated page background color to be consistent with navbar color during loading screen

### CODE REVIEW

**Review by the-simian — CHANGES_REQUESTED** [review]
I checked this out - make sure that if you're on a particular route, the nav item for that route is inert. (eg. if you're on home, clicking home should do nothing)

File: packages/website/components/navbar.js [hunk 1]
```diff
@@ -126,10 +126,12 @@ export default function Navbar({ bgColor = '', isLoggedIn, isLoadingUser }) {
           </a>
         </div>
         <div className={clsx("flex items-center", isSmallVariant ? 'justify-end' : '')} style={{ minHeight: 52 }}>
-          { !isSmallVariant && ITEMS.map(item => (
-            <a href={item.link} key={item.name} onClick={onLinkClick} className={clsx('text-sm text-w3storage-purple font-bold no-underline hover:underline align-middle', item.spacing)}>
-              { item.name }
-            </a>
+          {!isSmallVariant && ITEMS.map(item => (
+            <Link href={item.link} key={item.name} >
+              <a onClick={onLinkClick} className={clsx('text-sm text-w3storage-purple font-bold no-underline hover:underline align-middle', item.spacing)}>
```
Comment (redaphid) [inline packages/website/components/navbar.js:131]: Shouldn't the `onClick` be on the `<Link>`  element?

Comment (redaphid) [reply packages/website/components/navbar.js:131]: nvm

**Review by redaphid — APPROVED** [review]
Looks good to me!

**Review by the-simian — APPROVED** [review]
Navigation is much tighter now, much improved!

**Pull request merged**